### PR TITLE
[18.0][MIG] mgmtsystem_action: Migration to 18.0

### PR DESCRIPTION
**Pull request by Kimkhoi3010**

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]

**Review by TDu — APPROVED** [review]

File: mgmtsystem_action/views/mgmtsystem_action.xml [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+                <field name="reference" />
+                <field name="user_id" />
+                <field name="create_date" />
+                <field name="date_deadline" />
+                <field name="activity_ids" />
+                <field name="activity_state" />
+                <field name="message_needaction_counter" />
+                <templates>
+                    <field name="date_deadline" />
+                    <t t-name="kanban-box">
```
Comment (TDu) [inline mgmtsystem_action/views/mgmtsystem_action.xml]: @Kimkhoi3010  This line is preventing this module to be merged, could you fix it please
https://github.com/OCA/management-system/actions/runs/11903685874/job/33171125454#step:8:92
Thank you

Comment (Kimkhoi3010) [reply mgmtsystem_action/views/mgmtsystem_action.xml]: Sure, thanks for your time!

**Pull request merged**